internal/credential/vault/repository_credential_store.go

```diff
@@ -499,6 +487,50 @@ func (ps *privateStore) TableName() string {
 	return "credential_vault_store_private"
 }
 
+// applyUpdate takes the new and applies it to the orig using the field masks
+func applyUpdate(new, orig *CredentialStore, fieldMaskPaths []string) *CredentialStore {
```

Comment:
Why not make this a method on `*CredentialStore`?

Reply:
Done, thanks!